src/styles/getModalStyles/index.android.js

```diff
@@ -0,0 +1,8 @@
+import CONST from '../../CONST';
+import getModalStyles from './getModalStyles';
+
+export default getModalStyles({
+    shouldModalAddTopSafeAreaPadding: {
+        [CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED]: true,
```

Comment:
We only need to set `CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED` to `true` since that's the only thing that doesn't no break on android.